_posts/2019-03-17-graze-x-shopify.md

```diff
@@ -0,0 +1,118 @@
+
+- **Apps** - Shopify Apps are easy to install, some are just one click, and most paid apps offer a trial period. This would open an exciting path for our Digital Product team to rapidly test new site features and gain valuable learning before committing to development. We were able to test a wide range of apps and identified several that would immediately save development time:
+  - Shopify Reviews - comments and star ratings on products
+  - Zendesk - for integration with our Zendesk helpdesk
+  - PixelPop - for site-wide modal messaging
+  - Instafeed - for embedding an instagram feed
+
+- **CMS** - Apart from the homepage, we found CMS is quite limited unless you want use one of the page builder apps or the [section hack](https://oak.works/blog/technical/2017/03/23/shopify-sections-editor/). We didn't expect to be building many custom pages on Shopify, so this was not a blocker.
+
+- **Costs** - Our initial cost estimates had not factored in that we would be running multiple Shopify stores, increasing the cost of some of the paid-for apps that we were looking at. Despite this, we were still looking at substantial savings.
+
+- **Documentation** - Shopify API documentation is well maintained, and kept up to date. The Shopify forums however contain lots of outdated posts from many years ago (5+ years) that contain out of date information, making it hard to find
+
+- **Payment Gateways** - There are lots of payment gateway options available, including our existing payment providers (Adyen and Paypal). We also have the option of using [Shopify Payments](https://help.shopify.com/en/manual/payments/shopify-payments), which makes other payment method integrations (e.g. Amazon Pay, Apple Pay, Google Pay) available without any development work.
+
+- **Product Management** - Shopify Product management is straightforward, with a sensible API and intuitive naming. However, when looking at ways to store the legally required nutritional data against a product (allergens, nutritional information, ingredients), our options were limited:
+  - As part of the description - poor for managing, and editable by any merchandiser
+  - As images - poor for accessibility and again, editable by any merchandiser
+  - As [metafields](https://help.shopify.com/en/manual/products/metafields) - this was our only real option, metafields would allow us to store JSON however we encountered issues with character limits
+
+If we wanted to use Shopify we would need to find a different solution to managing product nutritional information.
+
+- **Single Sign On** - Single sign on would be possible using multipass and we still be able to master accounts on our subscription website. This would make things considerably less risky, as we would not need to migrate any customer data, part of the reason we could even consider this move is because we had chosen not to master customer data in Magento.
```

Comment:
```suggestion
- **Single Sign On** - Single sign on would be possible using multipass and we would still be able to master accounts on our subscription website. This would make things considerably less risky, as we would not need to migrate any customer data. Part of the reason we could even consider this move is because we had chosen not to master customer data in Magento.
```